src/services/security/modules/RedefineModule/index.ts

```diff
@@ -1,6 +1,9 @@
 import { REDEFINE_API } from '@/config/constants'
+import { normalizeTypedData } from '@/utils/web3'
+import { isObjectEIP712TypedData } from '@safe-global/safe-apps-sdk'
```

Comment:
Is the implementation of `isObjectEIP712TypedData` complex?
I would like to avoid depending on the apps SDK outside of the Safe Apps functionality.

Reply:
It's a one liner. I'll copy it into our utils.